en/sql/tuning.rst

```diff
@@ -688,6 +689,12 @@ The following hints can be specified in **UPDATE**, **DELETE** and **SELECT** st
     
         Hash List scan only works for predicates having a equal operation and does NOT work for predicates having OID type.
 
+*   **NO_LOGGING**: This is a hint not to include the redo in the log generated when inserting, updating, or deleting records to a table.
+
+    .. note::
+
+        Currently, The NO_LOGGING hint only affects the log created from the heap file when inserting, updating, or deleting records to a table. Therefore, there may be a problem that the data of the table and the index are inconsistent after recovery and the problem that the committed record is not recovered, etc. You should use it carefully.
```

Comment:
the part  ' and the problem that the committed record is not recovered...' might be vague, so I suggest changing the sentence to the following: 

```suggestion
        Currently, The NO_LOGGING hint only affects the log created from the heap file when inserting, updating, or deleting records to a table. Therefore, problems such as the inconsistency between the data of the table and the date of the index might occur after recovery; and the situation of committed record cannot be recovered might also occur, etc. You should use it carefully.
```

Reply:
@charischau I've already committed your proposal, but I'll have to change it to 'data of the index' instead of 'date of the index'.

Reply:
okay, 'date of the index' was a typo, I apologise.